src/Surfnet/Stepup/Identity/Identity.php

```diff
@@ -127,9 +128,9 @@ class Identity extends EventSourcedAggregateRoot implements IdentityApi
     private $vettedSecondFactors;
 
     /**
-     * @var RegistrationAuthority
+     * @var RegistrationAuthorityCollection|RegistrationAuthority[]
```

Comment:
Nitpick, is it actually possible to set an array of RegistratoinAuthority here? Or is the Collection class always used? In that case, I would prefer the removal of `RegistrationAuthority[]`